README.md

```diff
@@ -1,34 +1,46 @@
 # Mifos Self Service App
 
 ## Getting started
-You can follow this README or you can follow this [video](https://mifosforge.jira.com/wiki/spaces/docs/pages/498794497/Developer+Environment+Setup+for+Online+Banking+App+2.0).
 
 Clone the project:
 
-    $ git clone https://github.com/openMF/web-self-service-app.git
+    $ git clone https://github.com/gauravsaini03/self-service.git
 
 Install dependencies:
+Globally make sure you have the following node v8.9.4 this comes with npm 5.5.1 or above then run commmand below
+For exampla Do this by installing npm using nvm
+    $ nvm install v8.9.4
+   
+Make sure you are running npm v8.9.4 ( you might want to run command below to be sure)
+    $ nvm use 8.9.4 
+    $ nvm alias default 8.9.4
```

Comment:
Please educate me on nvm vs npm

Reply:
@avikganguly01 , 
nvm is a node version manager 
nvm manages node.js and npm versions. It’s designed to be installed per-user on bother windows and Linux its good for switching in between versions npm